File: origanboh_spring/src/main/java/com/generation/origanboh_spring/entities/AcquistoDettagliato.java
package com.generation.origanboh_spring.entities;


import lombok.Data;

@Data
public class AcquistoDettagliato extends Entity{
    private int id;
    private int idCliente;
    private int idProdotto;
    private String dataacquisto;
    private int quantita;
    private String nomeprodotto;
    private double prezzo;
    private int stock;
    private String descrizione;
    private String imgSource;
    private String autore;
    private String genere;
}

File: origanboh_angular/src/models/AcquistoDettagliato.ts
export interface AcquistoDettagliato {
  id: number;
  idCliente: number;
  idProdotto: number;
  dataacquisto: string;
  quantita: number;
  nomeprodotto: string;
  prezzo: number;
  stock: number;
  descrizione: string;
  imgSource: string;
  autore: string;
  genere: string;
}
